README.md

```diff
@@ -15,28 +15,26 @@ yarn install
 yarn dev
 ```
 
-To clean builds:
+Serves from <http://localhost:8000/>.
 
-```sh
-yarn clean
-```
-
-Serves from <http://localhost:1234>.
+If you want to a specific port, like 1234, just add `--port 1234`.
 
 ### Production
 
 ```sh
 yarn build
 ```
 
-Built files can be found in `/dist`.
+Built files can be found in `public/`.
 
-An easy way to try running the built files:
+An easy way to test out the built files:
 
 ```sh
-bash -c "cd dist && npx serve"
+yarn gatsby serve
```

Comment:
Let's add a command to `package.json` that wraps around this call, e.g. `yarn run-build` or something with an _even better name_

Reply:
I want the commands to be separate because you might want to kill the server but restart it later without recompiling everything. Would it be okay to have `yarn gatsby serve` as `yarn serve` or `yarn serve-built`?

I guess we can also have `yarn build-and-serve` as a wrapper around both commands. I just want to make the individual commands very discoverably.

Reply:
Oh that's what I had in mind, sorry for the confusion from my bad name. I don't want the commands to be clobbered together by any means.